docs/_source/docs/stack_group_config.rst

```diff
@@ -185,6 +185,17 @@ For example, if you wanted the ``dev`` StackGroup to build to a different
 region, this setting could be specified in the ``config/dev/config.yaml`` file,
 and would only be applied to builds in the ``dev`` StackGroup.
 
+Inheritance Strategy Override
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The inheritance strategy of some properties may be overridden by the stack group config.
+
+Strategy options:
+
+ - ``list_join``: Child configs are appended to parent configs.
+ - ``dict_merge``: Child configs will be merged with parent configs, the child keys taking precedence.
+ - ``child_wins``: Overrides parent if set.
```

Comment:
I chose to use the function names directly as the options, but it's possible a more "user friendly" might be better.

Reply:
Yes something like this, although perhaps `merge` instead of `prepend` as I think it will differentiate it better from `append`.

I'm happy to change this over if it's preferred?

Reply:
i think i would prefer friendlier names, however I'm not sure what that would look like exactly.  do we use `merge` instead of `list_join` and `dict_merge`?  if so that might be more confusing since we would be using one word to describe something similar but subtly different?

Reply:
I do like that, it seems like a clean UX.  Personally I don't see the dependency in behaviour to be much of a problem - as long as the user has a basic understanding of YAML, because lists and maps simply work differently when combined.

It feels like most words we could use would have the same problem? i.e most common words here conceptually apply to both operations.

Reply:
ok, lets go with `merge` for both list_join and dict_merge then. thanks

Reply:
I've updated this in the latest commit.